CodeStarterCamp_Week1/iceCream.swift

```diff
@@ -0,0 +1,12 @@
+for _ in 1...8{
```

Comment:
스위프트에서는 코드블럭을 나타내는 중괄호는 한칸 띄어씁니다!
```suggestion
for _ in 1...8 {
```